README.md

```diff
@@ -18,7 +18,7 @@ The Operator to install and manage the lifecycle of the [Kuadrant](https://githu
       * [If you are an <em>API Provider</em>](#if-you-are-an-api-provider)
       * [If you are a <em>Cluster Operator</em>](#if-you-are-a-cluster-operator)
 * [User guides](#user-guides)
-* [<a href="/doc/rate-limiting.md">Kuadrant Rate Limiting</a>](#kuadrant-rate-limiting)
+* [<a href="doc/rate-limiting.md">Kuadrant Rate Limiting</a>](#kuadrant-rate-limiting)
```

Comment:
Just for curiosity, is `/doc/rate-limiting.md` wrong? the link seems to be working (on Linux/Firefox)

Reply:
The leading slash breaks it in https://docs.kuadrant.io/kuadrant-operator/. Not sure if using the simple relative path will solve it tho.

Reply:
It works but it is broken in the website for the above in the table of contents ^^ . I am also not sure will this change fix it since it links to https://docs.kuadrant.io/doc/rate-limiting.md vs https://docs.kuadrant.io/kuadrant-operator/doc/rate-limiting/ which seems to be the correct url instead 🤷 

Though this change is also to give consitency to references to other docs in the repo. In most other links, we use `docs/x` instead of `/docs/x` 🤔